instrumentation-annotations-incubator/src/main/java/io/opentelemetry/instrumentation/annotations/Counted.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.annotations;
```

Comment:
probably this should be under `.incubator` package similar to `instrumentation-api-incubator` artifact and package structure?
```suggestion
package io.opentelemetry.instrumentation.annotations.incubator;
```

Reply:
Also moved the instrumentation implementation to incubator package and renamed the instrumentation module